configure

```diff
@@ -1095,6 +1095,11 @@ envopt CPP
 envopt CFLAGS
 envopt CXXFLAGS
 
+# stdc++ name in use
+# used to manage non-standard name (on OpenBSD for example)
+CFG_STDCPP_NAME=$(echo "stdc++" | sed "$($CFG_CC -v 2>&1 | sed -n "s/.*--program-transform-name='\([^']*\)'.*/\1/p")")
```

Comment:
`program-transform-name` is a sed regex (under OpenBSD for gcc-4.9 it is `s,^,e,`).

basically, it search for the configure flag `program-transform-name` in `gcc -v`, and use that with `sed` on the `"stdc++"` name. If the flag isn't found, the transformation applied is `sed ""` which means "no transformation".